docs/api/new-ref-doc-JDN.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+---
+
+# Revise a task due date
+
+Updates the `due_date` of an existing task. This can be useful when the task's deadline needs to be adjusted.
+
+## URL
+
+```shell
+{base_url}/tasks/{id}
+```
+
+## Method
+`PATCH`
+
+## Params
+
+| Parameter name | Type   | Description |
+| -------------- | ------ | ----------- |
+| `id`           | number | The record ID of the task to update |
+| `due_date`     | string | The date in ISO 8601 format         |
+
+## Request headers
+
+| Header name    | Type   | Description                           |
+| -------------- | ------ | ------------------------------------- |
+| `Content-Type` | string | Must be set to `application/json`     |
+
+## Request body (using cURL)
+
```

Comment:
It seems like the request body property from earlier should appear here and the following example should be labeled as an example.

Reply:
Okay. Did that.